### PR TITLE
2304-grant-request-page-translation

### PR DESCRIPTION
**Pull request by paulmsmith**
## Proposed changes

[Ticket](https://trello.com/c/AHsylvPl/2304-abstract-strings-from-your-grant-request-page)

### CODE REVIEW

File: app/views/funding_application/gp_project/grant_request/show.html.erb [hunk 1]
```diff
@@ -2,47 +2,49 @@
...
+    <%= t('views.funding_application.common.about_your_project') %>
+  </span>
+  <%= t('application.grant_request.page_heading') %>
 </h1>
 
 <% unless @grant_request_is_valid %>
   <div class="nlhf-panel nlhf-alert nlhf-alert--error nlhf-alert--slim govuk-!-margin-bottom-8">
     <h2 class="govuk-heading-m">
-      We have found a problem
+      <%= t('generic.there_is_a_problem') %>
```
Comment (ptrelease) [inline app/views/funding_application/gp_project/grant_request/show.html.erb:19]: There is a problem

File: app/views/funding_application/gp_project/grant_request/show.html.erb [hunk 2]
```diff
@@ -2,47 +2,49 @@
...
-          Your grant request is higher than &pound;10,000 which the maximum allowable for this funding programme.
+          <%= t('application.grant_request.text.8').html_safe %>
         <% end %>
       <% end %>
 
     </p>
 
     <p class="govuk-body govuk-!-font-size-19">
-      Please review your project costs and cash contributions.
+      <%= t('application.grant_request.text.9').html_safe %>
```
Comment (ptrelease) [inline app/views/funding_application/gp_project/grant_request/show.html.erb:47]: Do we need html safe on this one?

Comment (ptrelease) [reply app/views/funding_application/gp_project/grant_request/show.html.erb:47]: Also consider if we need on the subsequent translations in this file.

**Comment by paulmsmith** [comment]
Good spots on those @ptrelease - Done!

File: config/locales/en.yml [hunk 3]
```diff
@@ -23,6 +23,7 @@ en-GB:
     select_one_option: "Select one option"
     delete: "Delete"
     there_is_a_problem: "There is a problem"
+    we_have_found_a_problem: "we have found a problem"
```
Comment (ptrelease) [inline config/locales/en.yml:26]: Make an uppercase W on We?

**Review by ptrelease — APPROVED** [review]

**Pull request merged**